packages/website/content/docs-technical.md

```diff
@@ -68,7 +68,6 @@ Publish a Cannon package to the registry for all networks where this package has
 **Options**
 
 - `--chain-id` - The network which the registry to deploy to is on. Default: 1 (mainnet)
-- `--provider-url` - RPC endpoint to use for publishing. If not specified, uses a local wallet provider (such as Frame)
```

Comment:
This piece in the docs was incorrect, after looking through the publish.ts file there was no reference to use of the "provider-url" param